lib/request.js

```diff
@@ -1,4 +1,9 @@
 var request = require('superagent');
+require('superagent-proxy')(request);
+
+var checkProxy = function(req, proxy) {
+	return proxy ? req.proxy(proxy) : req;
```

Comment:
I've tried to google other way around this, but looks like ``superagent-proxy`` can't handle empty proxy with ``req.proxy``, so this approach seems 👍